exporter/datasetexporter/README.md

```diff
@@ -33,6 +33,8 @@ If you do not want to specify `api_key` in the file, you can use the [builtin fu
 - `traces`:
   - `aggregate` (default = false): Count the number of spans and errors belonging to a trace.
   - `max_wait` (default = 5s): The maximum waiting for all spans from single trace to arrive; ignored if `aggregate` is false.
+- `logs`:
+  - `export_resource_info_on_event` (default = false): Include resource info to DataSet Event while exporting Logs.
```

Comment:
This is already described in config.go, but sure good idea to do it also here

Reply:
Ideally table with options in the readme would be auto-generated from code to avoid duplication, things getting out of sync, etc. 

But that can be done in the future.